pysat/utils/files.py

```diff
@@ -26,14 +26,14 @@ def process_parsed_filenames(stored, two_digit_year_break=None):
     Parameters
     ----------
     stored : collections.orderedDict
-        Dict produced by `parse_fixed_width_filenames` or
-        `parse_delimited_filenames`
+        Ordered dictionary produced by `parse_fixed_width_filenames` or
```

Comment:
data, time, and version is the primary info parsed, but the functions also parse out additional items as provided by the user.

Reply:
Looking at the code it seems that only "year", "month", "day" (of year or of month, depending on whether or not month is included), "hour", "minute", "second" (of day or of minute, depending on whether hours and minutes are included or not), "version", "revision", and "cycle".  I was grouping "version", "revision", and "cycle" as types of version identifiers.  However, it seems that since these are the only supported data to be parsed this should also be listed in the docstring.  Is this correct?

Reply:
The listed keywords are the specific ones that are parsed out and used for sorting. The code will also parse out any other values in the string template but those aren't used for file sorting.

For example, this is around line 283 in `parse_delimited_filenames`
```
    # Parse format string to get information needed to parse filenames
    search_dict = construct_searchstring_from_format(format_str, wildcard=False)

    # Add non-standard keys
    for key in search_dict['keys']:
        if key not in stored:
            stored[key] = None
```

`stored` is initially defined as just the keys you list, 'year', 'month', etc., but then any non-standard keys identified by `construct_searchstring_from_format` are added.

Reply:
Can you provide a use example?

Reply:
This an example from older COSMIC files, 
```
pysat.Files.from_os(data_path, '{year2:04d}.{day2:03d}/ionPrf_C{sat_id:03d}.{year:04d}.{day:03d}.{hour:02d}.{minute:02d}.G{code:02d}_{year3:04d}.{radio:04d}_nc')
```

Comes from this pull https://github.com/pysat/pysat/pull/982